sdk/storage/azblob/blockblob/client_test.go

```diff
@@ -771,6 +765,11 @@ func (s *BlockBlobRecordedTestsSuite) TestStageBlockWithGeneratedCRC64() {
 func (s *BlockBlobRecordedTestsSuite) TestStageBlockWithCRC64() {
 	_require := require.New(s.T())
 	testName := s.T().Name()
+	var b [8]byte
+	_, err := rand.Read(b[:])
+	if err != nil {
+		return
+	}
```

Comment:
```suggestion
	_require.NoError(err)
```

Reply:
Can you also move this to the place where its used

Reply:
this is for solving the lint errors